index.html

```diff
@@ -30,6 +30,130 @@ <h1>Frehiywot <br />Nega</h1>
     </p>
     <a href="#start_collaboration" class="collaboration">Start collaboration</a>
     <br>
-    <div class="image_geometry_2"></div> 
+    <div class="image_geometry_2"></div>
+    <h2>Projects</h2>
+  </section> 
+
+  <section class="Works-section" >
```

Comment:
- [ ] Kindly use `display: grid` on this section.
_You have done it for the cards, but it should be on the main container_